src/AgOledDisplay.cpp

```diff
@@ -307,20 +307,21 @@ void OledDisplay::showDashboard(const char *status) {
       int pm25 = value.pm25_1;
       if (config.hasSensorSHT) {
         pm25 = ag->pms5003.compensated(pm25, value.Humidity);
+        logInfo("PM2.5:" + String(value.pm25_1) + String("Compensated:") + String(pm25));
       }
       DISP()->setFont(u8g2_font_t0_22b_tf);
       if (config.isPmStandardInUSAQI()) {
-        if (utils::isValidPMS(value.pm25_1)) {
-          sprintf(strBuf, "%d", ag->pms5003.convertPm25ToUsAqi(value.pm25_1));
+        if (utils::isValidPMS(pm25)) {
```

Comment:
the method name isValidPMS is misleading, propose to rename to simply isValid()

Reply:
it's the API on `utils` can not use `isValid` cause there are multiple API check sensor valid like `CO2`, `Temperature`, etc.

Reply:
OK, then I propose to write

isValidPm()

Reply:
@nick-4711 
Updated 753f22923caa97cf0630c5a8da1ec6c6f44cb47e